.github/workflows/quarto-publish.yml

```
@@ -0,0 +1,29 @@
+on:
```

I think this should address the TOKEN permissions issue.

```suggestion
permissions:
  contents: write

on:
```